### ISSUE TITLE
base_local_planner: rotateToGoal() can generate too-small angular velocity

### ISSUE THREAD

**Issue by hershwg** [ros-planning/navigation#1]
If a low yaw acceleration limit is given, then TrajectoryPlannerROS::rotateToGoal() can generate a yaw velocity below the user-specified minimum for in-place rotation (trajectory_planner_ros.cpp:305):

``` c++
//we also want to make sure to send a velocity that allows us to stop when we reach the goal given our acceleration limits
    double max_speed_to_stop = sqrt(2 * acc_lim_theta_ * fabs(ang_diff));

    v_theta_samp = sign(v_theta_samp) * std::min(max_speed_to_stop, fabs(v_theta_samp));
```

The planner should never violate the minimum in-place rotation velocity. The result might be overshoot of and oscillation about the desired angle. That's ok; it's up to the user to trade off three parameters: max yaw acceleration, min in-place rotation velocity, and yaw goal tolerance.

**Comment by hershwg** [comment]
Workaround: set min_x_vel > 0 and min_trans_vel > 0.  min_rot_vel is only enforced when translation speed  is less than its minimum.

**Comment by AchmadFathoni** [comment]
Why this issue still open?

**Comment by aaronhoy** [comment]
This appears to have been fixed by #725.  Closing.
